webapp/src/components/propertyValueElement.tsx

```diff
@@ -56,6 +56,58 @@ const PropertyValueElement = (props:Props): JSX.Element => {
         }
     }
 
+    if (propertyTemplate.type === 'multiSelect') {
```

Comment:
It would be cleaner to have an individual React component for each property type, like I did here - https://github.com/mattermost/focalboard/pull/393

It's a team call though.

Reply:
You guys can make the call, but my advice is to refactor later. i.e. either check in new functionality, or refactor existing functionality. If the change is relatively small or contained, then go ahead and refactor now, otherwise my advice is to do it in a subsequent commit.

Reply:
Done